stable/prometheus-pushgateway/templates/deployment.yaml

```diff
@@ -1,12 +1,12 @@
+{{- $labelChart := include "prometheus-pushgateway.chart" $ -}}
+{{- $labelApp := include "prometheus-pushgateway.name" $ -}}
```

Comment:
please move this to the _helper.tmpl, ex https://github.com/helm/charts/blob/master/stable/prometheus-node-exporter/templates/_helpers.tpl#L27-L36